ddtrace/internal/telemetry/writer.py

```diff
@@ -124,6 +128,10 @@ def __init__(self, interval):
         self._forked = False  # type: bool
         self._events_queue = []  # type: List[Dict]
         self._lock = forksafe.Lock()  # type: forksafe.ResetObject
+        # Currently telemetry only supports reporting a single error.
+        # If we'd like to report multiple errors in the future
+        # we could hack it in by xor-ing error codes and concatenating strings
+        self._error = (0, "")  # type: Tuple[int, str]
```

Comment:
What are examples of different error codes? What do the different values mean? 
Can we link to the instrumentation telemetry spec here: https://github.com/DataDog/instrumentation-telemetry-api-docs/blob/main/GeneratedDocumentation/ApiDocs/v2/SchemaDocumentation/Schemas/error.md

Reply:
We're currently only going with two error codes, 1 means error, 0 means no error. We can implement other error codes in the future for telemetry v2 work.